docs/concepts/ipns.md

```diff
@@ -5,23 +5,25 @@ description: Learn about mutability in IPFS, InterPlanetary Name System (IPNS),
 
 # InterPlanetary Name System (IPNS)
 
-- [Mutability in IPFS](#mutability-in-ipfs)
```

Comment:
I'm not clear from the diff, what changed on this page?

Reply:
`keyholder` -> `key holder`